content/en/platform/corda/4.10/enterprise/node/operating/monitoring-and-logging/opentelemetry.md

```diff
@@ -0,0 +1,111 @@
+
+* Include the `corda-opentelemetry-driver` into the drivers directory of the Corda node.
+* Start the Corda node with an OpenTelemetry Java agent installed.
+
+## Using the OpenTelemetry driver
+
+To use the OpenTelemetry driver, copy `corda-opentelemetry-driver-%VERSION%.jar` into the drivers directory of the Corda node. This driver can be downloaded from Artifactory and is included in the release pack.
+
+## Using the OpenTelemetry Java agent
+
+To use the OpenTelemetry Java agent:
+
+1. Download the agent from [GitHub](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases). The file you need is `opentelemetry-javaagent.jar`.
+2. Start the Corda node with the following example command line: `java -Dcapsule.jvm.args=“-javaagent:/PATH-TO-OT-JAVAAGENT/opentelemetry-javaagent.jar -Dotel.service.name=YOUR-SERVICE-NAME” -jar corda.jar`.
+3. Replace `PATH-TO-OT-JAVAAGENT` with the full path to where you placed the OpenTelemetry Java agent.
+4. Replace `YOUR-SERVICE-NAME` with the service name you would like the Corda node to be identified as.
+
+{{< note >}}
+This method of linking the SDK with the Corda node enables OpenTelemetry automated instrumentation, which means some other third-party libraries included with Corda will also generate spans. See the [OpenTelemetry](https://opentelemetry.io/docs/concepts/instrumenting-library/) website for details on which libraries are enabled.
+{{< /note >}}
+
+
+## Setting up OpenTelemetry
+
+Corda has been tested with the OpenTelemetry collector and a Jaeger backend. Refer to the [OpenTelemetry](https://opentelemetry.io/docs/collector/) and [Jaeger](https://www.jaegertracing.io/) websites for details on how to setup a collector and Jaeger backend.
+
+Corda OpenTelemetry can also be enabled or disabled via a node configuration parameter, such as `telementry.openTelemetryEnabled = true`.
+
+If this setting is true and the OpenTelemetry SDK has been linked to Corda as described above, the node will generate spans. If this setting is false, the node won't generate spans, even if the node is linked with the SDK.
+
+## Client API
+
+OpenTelemetry span generation is also incorporated into the RPC client API. If you want your Corda client code to generate spans, you need to include the OpenTelemetry SDK as a dependency. The methods described above only apply to the Corda node. 
+
+<!-- (TODO: Give example of dependencies needed). -->
+
+## Creating your own Spans
+
+The OpenTelemetry API may be used in your flows and in your client code to create spans and baggage. To get an instance of the OpenTelemetry API in a flow, make the following call:
+
+```kotlin
+val openTelemetry: OpenTelemetry? = serviceHub.telemetryService.getTelemetryHandle(OpenTelemetry::class.java)
+    }
+``` 
+
+From the client API, where RPC is a CordaRPCConnection, you would use the following:
+
+```kotlin
+val openTelemetry: OpenTelemetry? = rpc.getTelemetryHandle(OpenTelemetry::class.java)
+    }
+``` 
+
```

Comment:
Once you have the open telemetry handle then you may call the open telemetry api as documented in the open telemetry documentation.